go.mod

```diff
@@ -15,11 +15,13 @@ require (
 	github.com/hashicorp/go-cleanhttp v0.5.2
 	github.com/hashicorp/go-hclog v1.4.0
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/hashicorp/vault/api v1.8.3
-	github.com/hashicorp/vault/sdk v0.7.0
+	github.com/hashicorp/vault/api v1.9.0
+	github.com/hashicorp/vault/sdk v0.8.1
 	golang.org/x/oauth2 v0.4.0
 )
 
+require github.com/hashicorp/go-kms-wrapping/v2 v2.0.7 // indirect
```

Comment:
We could probably add this to the `require` block below and run `go mod tidy`. Not sure why Go added this separately 🤔